components/form/ErrorMessage.js

```diff
@@ -11,10 +11,11 @@ export function ErrorLabel({ errorClassName, name }) {
     </>
   );
 }
-ErrorLabel.PropTypes = {
-  name: PropTypes.string
+ErrorMessage.propTypes = {
+  name: PropTypes.string,
+  errorClassName: PropTypes.string
 };
 
-ErrorLabel.defaultProps = {
+ErrorMessage.defaultProps = {
   name: null
```

Comment:
Since the `errorClassName` is not `Required`, then it is best to define a defaultProps for it too as you have done for `name`.